src/CoreMidi/MidiServices.cs

```diff
@@ -1542,6 +1544,15 @@ public int Add (string name, bool embedded, nuint numSourceEndpoints, nuint numD
 			}
 		}
 
+		[TV (13,0), Mac (10,15), iOS (13,0)]
```

Comment:
why `[TV]` ? it was not reported (or updated) in the xtro files

Reply:
But it i present in the headers, from the diff:

extern const CFStringRef    kMIDIPropertyNameConfigurationDictionary            API_AVAILABLE(macos(10.15), ios(13.0), tvos(13.0));

Reply:
Headers can lie ;-)
The framework is not available on tvOS, so the API can't be present.